system-tests/projects/e2e/cypress/support/util.js

```diff
@@ -70,12 +56,6 @@ export const verify = (ctx, options) => {
       .should('not.include.text', '__stackReplacementMarker')
 
       cy.contains('.runnable-err-stack-trace .runnable-err-file-path', openInIdePath.relative)
```

Comment:
I left the assertions for the element presence but removed the action assertion. I struggled to find a good way to continue to have coverage here; any suggestions are welcomed.